.rubocop.yml

```diff
@@ -5,6 +5,9 @@ AllCops:
   DisplayCopNames: true
   Include:
     - '**/Rakefile'
+  Exclude:
+    - 'spec/internal/bin/*'
```

Comment:
I thought it might be good to catch bad stuff generated into spec/internal by the generators.